src/ckeditor/ckeditor.component.ts

```diff
@@ -251,11 +253,19 @@ export class CKEditorComponent implements AfterViewInit, OnDestroy, ControlValue
 
 	ngAfterViewInit(): void {
 		getEditorNamespace( this.editorUrl ).then( () => {
+			// Check if component instance was destroyed before `ngAfterViewInit` call (#110).
+			// Here, `this.instance` is still not initialized and so additional flag is needed.
+			if( this._destroyed ) {
```

Comment:
I think the code style is wrong here 😇
```suggestion
			if ( this._destroyed ) {
```

Reply:
fixed thx